chart/templates/_helpers.tpl

```diff
@@ -182,6 +182,8 @@ env:
   value: "https://{{ $gp.hostname }}"
 - name: GITPOD_REGION
   value: {{ $gp.installation.region | quote }}
+- name: GITPOD_WS_DNS_LABEL
+  value: "ws-{{ $gp.installation.cluster }}"
```

Comment:
we should not rely on this field, only `hostname`.

Reply:
Hi Anton, thank you for the heads up! I will take a look at this tomorrow and share an update. I need to look at the code some more.